Sources/SnowManLogic/ViewLogic/MockUI.swift

```diff
@@ -0,0 +1,13 @@
+public class MockUI: Receivable {
+	public var getUserInputReturn = ""
```

Comment:
We discussed a preferred coding style for handling optional variables, and that empty variables are a code smell
```swift
public var foo: Type?
```
or
```swift
public var foo: Type!
```